docs/sources/configure-client/grafana-alloy/_index.md

```diff
@@ -81,6 +81,18 @@ In pull mode, the collector periodically retrieves profiles from Golang applicat
 1. Ensure Alloy is set to pull mode and targeting the correct pprof endpoints. For step-by-step instructions, visit the [Go (Pull Mode)](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/grafana-alloy/go_pull) documentation.
 1. The collector queries the pprof endpoints of your Golang application, collects the profiles, and forwards them to the Pyroscope server.
 
+## Receiving profiles from Pyroscope SDKs
+
+Alloy (not available in Grafana Agent) can receive profiles from applications instrumented with Pyroscope SDKs through the `pyroscope.receive_http` component. This approach provides several key advantages:
+- Improved performance by sending profiles to a local Alloy instance instead of over the internet to Grafana Cloud, reducing latency and application impact
+- Separation of infrastructure concerns from application code - developers don't need to handle authentication, tenant configuration, or infrastructure labels in their code
+- Centralized management of authentication and metadata enrichment (for example, Kubernetes labels, business labels)
```

Comment:
We try to avoid using parenthetical statements. It felt like having the Agent statement up front took away from the focus on Alloy. I've added a statement below noting that the capability isn't available in the legacy Grafana Agent. What do you think?
```suggestion
Alloy can receive profiles from applications instrumented with Pyroscope SDKs through the `pyroscope.receive_http` component. This approach provides several key advantages:
- Improved performance by sending profiles to a local Alloy instance instead of over the internet to Grafana Cloud, reducing latency and application impact
- Separation of infrastructure concerns from application code - developers don't need to handle authentication, tenant configuration, or infrastructure labels in their code
- Centralized management of authentication and metadata enrichment (for example, Kubernetes labels, business labels)

This capability is not available in the legacy Grafana Agent.
```